owncloudData/src/test/java/com/owncloud/android/data/shares/datasources/OCRemoteShareDataSourceTest.kt

```diff
@@ -181,10 +231,23 @@ class OCRemoteShareDataSourceTest {
         assertEquals("Photos/img1.png", publicShareAdded.path)
         assertEquals(false, publicShareAdded.isFolder)
         assertEquals("http://server:port/s/112ejbhdasyd1", publicShareAdded.shareLink)
+
+        verify(exactly = 1) {
+            clientManager.getShareService(OC_ACCOUNT_NAME)
+            ocShareService.insertShare(
+                remoteFilePath = "Photos/img1.png",
+                shareType = com.owncloud.android.lib.resources.shares.ShareType.fromValue(ShareType.PUBLIC_LINK.value)!!,
+                shareWith = "",
+                permissions = 1,
+                name = "",
+                password = "",
+                expirationDate = 0
+            )
+        }
     }
 
     @Test
-    fun updatePublicShare() {
+    fun `updateShare update a public share returns OCShare`() {
```

Comment:
You should apply all the comments I add in the review to the whole class, in the previous CR I suggested changing the analogue method for private share this way:
```suggestion
    fun `updateShare for public share returns OCShare`() {
```

Reply:
Done